aspnetcore/fundamentals/host/hosted-services.md

```diff
@@ -12,6 +12,10 @@ uid: fundamentals/host/hosted-services
 
 By [Jeow Li Huan](https://github.com/huan086)
 
+:::moniker range="< aspnetcore-6.0"
+[!INCLUDE [not-latest-version](~/includes/not-latest-version.md)]
+:::moniker-end
+
 :::moniker range=">= aspnetcore-6.0"
```

Comment:
This will be easy to keep up to date with each new version.
cc @tdykstra @wadepickett @guardrex